src/TransferRestrictor.sol

```diff
@@ -28,6 +28,14 @@ contract TransferRestrictor is Ownable2Step, ITransferRestrictor {
     constructor(address owner) {
         _transferOwnership(owner);
     }
+    
+    /// ------------------ Geeters ------------------ ///
+    
+    /// @notice Check if `account` is restricted
+    /// @param account The account to check
+    function isBlacklisted(address account) external view returns (bool) {
```

Comment:
public variables automatically create getters. Just rename blacklist mapping.

Reply:
we are accessing transferRestrictor through ITransferRestrictor, not directly, that is why we need to create a getter `isBlacklisted`

Reply:
Yes, add `isBlacklisted` to `ITransferRestrictor`, but it will work to simply rename `blacklisted` -> `isBlacklisted` in `TransferRestrictor`